examples/AN00162_i2s_loopback_demo/Makefile

```diff
@@ -1,7 +1,14 @@
 # The TARGET variable determines what target system the application is
 # compiled for. It either refers to an XN file in the source directories
 # or a valid argument for the --target option when compiling
-TARGET = XR-AUDIO-216-MC
+XCOREAI ?= 0
+ifeq ($(XCOREAI),0)
+TARGET = XCORE-200-EXPLORER
+XCC_FLAGS = $(EXTRA_FLAGS) -O2 -report -g -DSIM_LOOPBACK_TEST=1 -DDEBUG_PRINT_ENABLE=1
+else
+TARGET = XCORE-AI-EXPLORER
+XCC_FLAGS_XCORE_AI = $(EXTRA_FLAGS) -O2 -report -g -DSIM_LOOPBACK_TEST=1 -DDEBUG_PRINT_ENABLE=1 -D__XS2A__
```

Comment:
I don't think the -D__XS2A__ define is needed since all the ifdef __XS2A__ have been changed to include __XS3A__.